### PR TITLE
file5

### PR DESCRIPTION
**Pull request by ghuttulavinay**
review this file

### CODE REVIEW

File: file5 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+- name: Registering container name
+  set_fact:
+    cname: "{{ cname_prefix ~ '-options' }}"
```
Comment (anirudh2151) [inline file5:4]: Why are you using this options?

Comment (anirudh2151) [reply file5:4]: What I'd the purpose of using Options??

File: file5 [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    cname_h1: "{{ cname_prefix ~ '-options-h1' }}"
+    cname_h2: "{{ cname_prefix ~ '-options-h2' }}"
+    cname_h3: "{{ cname_prefix ~ '-options-h3' }}"
+- name: Registering container name
```
Comment (anirudh2151) [inline file5:8]: Drive yes b there's @ghuttulavinay

File: file5 (hunk 1 above)
Comment (ghuttulavinay) [reply file5:4]: gshxhhdjdbd

**Review by anirudh2151 — APPROVED** [review]

**Pull request merged**